<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stage_widget</class>
 <widget class="QWidget" name="stage_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="left_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rel_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Relative Length (mm):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute X (mm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="abs_x_line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abs_move_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Relative Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Reference Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Reference Point</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_length_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Length (mm):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_length_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Length (mm):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Shots:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num_shots_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Shots:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="raster_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Raster</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_shots_lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="abs_lbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rel_lbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set_x_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set X Bound</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enable_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QLabel" name="messages">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Group Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="x_group_combo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="y_group_combo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="kill_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kill All</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Group Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initialize_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute Y (mm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="abs_y_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Width (mm):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_width_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set_y_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Y Bound</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Raster</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_min_line">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_min_line">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Travel (mm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_max_line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_max_line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Travel (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OPT. Rep Rate (s): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rep_rate_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>left_btn</zorder>
  <zorder>rel_line</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>abs_x_line</zorder>
  <zorder>label_3</zorder>
  <zorder>abs_move_btn</zorder>
  <zorder>label_4</zorder>
  <zorder>set_btn</zorder>
  <zorder>return_btn</zorder>
  <zorder>step_length_line</zorder>
  <zorder>label_5</zorder>
  <zorder>sample_length_line</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>num_shots_line</zorder>
  <zorder>label_8</zorder>
  <zorder>raster_btn</zorder>
  <zorder>label_9</zorder>
  <zorder>max_shots_lbl</zorder>
  <zorder>abs_lbl</zorder>
  <zorder>rel_lbl</zorder>
  <zorder>set_x_btn</zorder>
  <zorder>enable_btn</zorder>
  <zorder>messages</zorder>
  <zorder>label_10</zorder>
  <zorder>x_group_combo</zorder>
  <zorder>y_group_combo</zorder>
  <zorder>kill_btn</zorder>
  <zorder>label_11</zorder>
  <zorder>initialize_btn</zorder>
  <zorder>label_12</zorder>
  <zorder>abs_y_line</zorder>
  <zorder>label_13</zorder>
  <zorder>sample_width_line</zorder>
  <zorder>set_y_btn</zorder>
  <zorder>right_btn</zorder>
  <zorder>up_btn</zorder>
  <zorder>down_btn</zorder>
  <zorder>stop_btn</zorder>
  <zorder>x_min_line</zorder>
  <zorder>y_min_line</zorder>
  <zorder>label_14</zorder>
  <zorder>x_max_line</zorder>
  <zorder>y_max_line</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>rep_rate_line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
